admin/ajax.php

```diff
@@ -311,7 +336,7 @@ function wpseo_upsert_new( $what, $post_id, $new, $original ) {
 function wpseo_get_export() {
 	check_ajax_referer( 'wpseo-export' );
 
-	$include_taxonomy = ( WPSEO_Utils::filter_input( INPUT_POST, 'include_taxonomy' ) === 'true' ) ? true : false;
+	$include_taxonomy = ( filter_input( INPUT_POST, 'include_taxonomy' ) === 'true' ) ? true : false;
```

Comment:
`$include_taxonomy = ( filter_input( INPUT_POST, 'include_taxonomy' ) === 'true' );` will do the same